app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb

```diff
@@ -148,6 +148,10 @@ def validate_entitled_processors(_field, values, _dlg, _fld, value)
     end
   end
 
+  def validate_pin_policy(_field, _values, _dlg, _fld, value)
+    return _('VM pinning policy can only be none, soft, or hard') if not ['none', 'soft', 'hard'].include? value
```

Comment:
I think you want either:
`unless ['none', 'soft', 'hard'].include?(value)`
or
`if ['none', 'soft', 'hard'].include_none?(value)`

Reply:
there is exclude also.